src/tools/AccessibilityChecker.js

```diff
@@ -0,0 +1,47 @@
+const runner = require('../lib/runner.js');
+
+const resultDefaults = {
+  time: {
+    start: 0,
+    end: 0,
+    get duration() {
+      return this.end - this.start;
+    },
+  },
+  reports: [],
+};
+let results;
+
+/**
+ * @param {string} path The root path begin the scan in
+ * @param {boolean} [logResults=true] Whether or not to log the result to the console
```

Comment:
in the function signature the default is set to false

Reply:
Fixed